visidata/sheets.py

```diff
@@ -1052,6 +1052,7 @@ def updateColNames(sheet, rows, cols, overwrite=False):
 
 # when diving into a sheet, remove the index unless it is precious
 SheetsSheet.addCommand('g^R', 'reload-selected', 'for vs in selectedRows or rows: vs.reload()', 'reload all selected sheets')
+IndexSheet.addCommand('g^R', 'reload-selected', 'for vs in selectedRows or rows: vs.reload()', 'reload all selected sheets')
```

Comment:
The SheetsSheet is also an IndexSheet, so we probably just want to change SheetsSheet to IndexSheet.  Please then verify that g^R still works on the SheetsSheet.

Reply:
Agreed, just done this and tested with SheetsSheet 👍